src/components/CadenceEditor/ControlPanel/index.tsx

```diff
@@ -343,8 +351,25 @@ const ControlPanel: React.FC<ControlPanelProps> = (props) => {
   };
 
   const isOk = !haveErrors && validCode !== undefined && !!validCode;
-  let statusIcon = isOk ? <FaRegCheckCircle /> : <FaRegTimesCircle />;
-  let statusMessage = isOk ? 'Ready' : 'Fix errors';
+  let statusIcon;
+  let statusMessage;
+  switch (true) {
+    case isOk && showPrompt:
+      statusIcon = (
+        <FaExclamationTriangle style={{ color: theme.colors.warning }} />
```

Comment:
It's better to set color of this one here:
https://github.com/onflow/flow-playground/blob/d385f01c36591881c0e3e20c69068c0e39c7003f/src/components/Arguments/styles.tsx#L109-L124